content/ember-data/v3/default-serializer.md

```diff
@@ -0,0 +1,31 @@
+---
+id: ember-data:default-serializers
+title: Default serializers usage
+until: '4.0.0'
+since: '3.12.0'
+---
+## Default serializer derprecations
+### deprecate adapter.serializer and adapter.defaultSerializer fallbacks
```

Comment:
I am 100% confident that `adapter.serializer` was never a working codepath and we've now removed it. I think we can just say this is for `adapter.defaultSerializer`